cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -347,24 +321,16 @@ def restart_unit_services():
     call(['systemctl', 'restart', 'kube-proxy'])
 
 
-def get_kube_api_server(kube_api):
+def get_kube_api_servers(kube_api):
```

Comment:
We're doing this in a *lot* of cases. I wonder if the http interface shouldn't just have a formatted output method to give us this. We've implemented similar logic in a lot of other charms. I'm not opposed to having this here, i just feel like we're re-inventing the same inclined plane.